content/operate/rs/security/recommended-security-practices.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Recommended security practices
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: null
+linkTitle: Recommended security practices
+hideListLinks: true
+weight: 5
+aliases:
+    - /operate/rs/security/admin-console-security/
+---
+
+## Deployment security
+
+When deploying Redis Enterprise Software to production, we recommend the following practices:
+
+- **Deploy Redis Enterprise inside a trusted network**:  Redis Enterprise is database software and should be deployed on a trusted network not accessible to the public internet. Deploying Redis Enterprise in a trusted network reduces the likelihood that someone can obtain unauthorized access to your data or the ability to manage your database configuration.
+
+- **Implement anti-virus exclusions**: To ensure that anti-virus solutions that scan files or intercept processes to protect memory do not interfere with Redis Enterprise software, customers should ensure that anti-virus exclusions are implemented across all nodes in their Redis Enterprise cluster in a consistent policy. This helps ensure that anti-virus software does not impact the availability of your Redis Enterprise cluster.
+
+    If you are replacing your existing antivirus solution or installing/supporting Redis Enterprise, make sure that the below paths are excluded:
+
+    {{< note >}}
+For antivirus solutions that intercept processes, binary files may have to be excluded directly depending on the requirements of your anti-virus vendor.
+    {{< /note >}}
+
+    | **Path** | **Description** |
+    |------------|-----------------|
+    | /opt/redislabs | Main installation directory for all Redis Enterprise Software binaries |
+    | /opt/redislabs/bin | Binaries for all the utilities for command line access and managements such as "rladmin" or "redis-cli" |
+    | /opt/redislabs/config | System configuration files |
+    | /opt/redislabs/lib | System library files |
+    | /opt/redislabs/sbin | System binaries for tweaking provisioning |
+
+- **Send logs to a remote logging server**: Redis Enterprise is configured to send logs by default to syslog. To send these logs to a remote logging server you must [configure syslog]({{<relref "/operate/rs/clusters/logging/log-security">}}) based the requirements of the remote logging server vendor. Remote logging helps ensure that the logs are not deleted so that you can rotate the logs so that your server disk does not fill up.
```

Comment:
```suggestion
- **Send logs to a remote logging server**: Redis Enterprise is configured to send logs by default to syslog. To send these logs to a remote logging server you must [configure syslog]({{<relref "/operate/rs/clusters/logging/log-security">}}) based the requirements of the remote logging server vendor. Remote logging helps ensure that the logs are not deleted so that you can rotate the logs to prevent your server disk from filling up.
```